### PR TITLE
EM-338: Fixed file upload type and size limit for comment attachments.

### PR DESCRIPTION
**Pull request by esune**
No tests for this change, we will need to refactor how this module is structured before we tackle them, since all the functions seem to be DOM-manipulation related.

### CODE REVIEW

File: src/app/comment-period/comment-period.component.spec.ts [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
 import { FileSizePipe } from '../pipes/filesize.pipe';
+import { ObjectFilterPipe } from '../pipes/object-filter.pipe';
+import { OrderByPipe } from '../pipes/order-by.pipe';
 import { Api } from '../services/api';
+import { CommentPeriodService } from '../services/comment-period.service';
+import { CommentPeriodComponent } from './comment-period.component';
+import { PCPInfoModalComponent } from './pcp-info-modal/pcp-info-modal';
+import { SubmitCommentModalComponent } from './submit-comment-modal/submit-comment-modal';
+import { SubmitCommentProgressModalComponent } from './submit-comment-progress-modal/submit-comment-progress-modal';
+
```
Comment (NickPhura) [inline src/app/comment-period/comment-period.component.spec.ts]: Im gonna need you to fix these empty lines.

File: src/app/comment-period/submit-comment-modal/submit-comment-modal.ts [hunk 2]
```diff
@@ -47,8 +49,28 @@ export class SubmitCommentModalComponent implements OnInit {
     element.classList.add('error', 'form-control-danger');
   }
 
+  validateAttachments(files) {
```
Comment (NickPhura) [inline src/app/comment-period/submit-comment-modal/submit-comment-modal.ts]: Could throw some types in here.

File: src/app/comment-period/comment-period.component.spec.ts (hunk 1 above)
Comment (NickPhura) [reply src/app/comment-period/comment-period.component.spec.ts]: This is super important.

**Comment by esune** [comment]
Updated according to code review comments.
Please SQUASH the  merge commit, when you merge the change. ;-)

**Pull request merged**